Mobile Buy SDK/Mobile Buy SDK/Utils/BUYApplePayHelpers.m

```diff
@@ -51,11 +51,6 @@ @interface BUYApplePayHelpers ()
 
 @implementation BUYApplePayHelpers
 
-- (instancetype)initWithClient:(BUYClient *)client checkout:(BUYCheckout *)checkout
-{
-	return [self initWithClient:client checkout:checkout shop:nil];
-}
-
 - (instancetype)initWithClient:(BUYClient *)client checkout:(BUYCheckout *)checkout shop:(BUYShop *)shop
```

Comment:
Looks like we need to set `checkout` a few times, but the other properties can be made readonly.